hugegraph-server/hugegraph-api/src/main/java/org/apache/hugegraph/metrics/SlowQueryLog.java

```diff
@@ -19,25 +19,36 @@
 
 public class SlowQueryLog {
```

Comment:
to be removed?

Reply:
> to be removed?

depends on whether to use it for subsequent optimization(could keep it now)